person.rb

```diff
@@ -1,28 +1,35 @@
-require_relative 'nameable'
+require './nameable'
+require './decorator'
+require './rental'
 
 class Person < Nameable
-  attr_accessor :name, :age
   attr_reader :id
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
-    super()
-    @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super()
   end
```

Comment:
- [ ] In the Person class, `@rentals` should be initialized according to the `a person has many rentals` relationship. Check out [the example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) about constructors in a previous lesson.